src/components/workspace/CanvasOutput.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+
+/**
+ * Takes the output of the rendered graphics (in a hidden canvas tag under <body>)
+ * and makes it into a new <canvas> output for viewing.
+ */
+class CanvasOutput extends React.Component<{}, {}> {
+  private $canvas: HTMLCanvasElement | null
+  private $parent: HTMLElement | null
```

Comment:
Are `$variables` a convention for variables pointing to HTMLElements? If so I'll open an issue changing the other HTMLElement variable names.